### ISSUE TITLE
[UI] Enhance: Advanced Encryption --> Easy encryption

### ISSUE THREAD

**Issue by Echolon** [kriztan/Pix-Art-Messenger#227]
Hi,

as mentioned in the Chat, most users can neither diffentiate between all the encryption choice nor handle anything else than OMEMO really. This please switch the interface choice from

Unencrypted
OMEMO
OTR
OpenPGP

to

Unencrypted
Encrypted (OMEMO)

*For all advanced users* OTR and OpenPGP should be only added by opt-in from expert settings (!)

Cheers

**Comment by ForkedLightning** [comment]
I disagree with this strongly. OTR and OpenPGP is still used by many clients especially through non-XMPP gateways that do not and will not support the OMEMO protocol. It will be impossible to explain to less tech savvy users on Pix-Art the steps to start encryption if you hide it in some setting somewhere.

What can be done is to make the other options "less desirable". For example grouping OTR and OpenPGP and making the next smaller so OMEMO stands out, or adding helpful text "OMEMO (Default)" "OTR (Expert)".

**Comment by Echolon** [comment]
@ForkedLightning  All the experts will find the options in the Expert Settings to bring it back.

How will this make it harder to bring people to encryption? Less tech savvy users won't use OpenPGP nor OTR... - and if so, they definitely should be able to go to settings and bring it back. However, Pix-Art is made for people who have no knowledge about encryption at all. So it is right in my opinion to make it much easier for them by this.

However, you could still user Conversations Legacy

**Comment by Echolon** [comment]
Would be great to see this soon. I think this would even help most users regarding #263

**Comment by Echolon** [comment]
With respect to #427, may you directly combine the removal commit with this by June?
